delimiter-stream.js

```diff
@@ -1,54 +1,103 @@
-var Transform = require('stream').Transform;
-var util = require('util');
+'use strict';
 
+const Transform = require('stream').Transform;
+const util = require('util');
+const defaults = {
+  delimiter: '\n',
+  encoding: 'utf8'
+};
 
 util.inherits(DelimiterStream, Transform);
 
-function DelimiterStream(options) {
+function DelimiterStream (options) {
   Transform.call(this, options);
 
-  var defaults = {
```

Comment:
What is the reason for moving the defaults out of the class definition?

Reply:
I moved it outside of the constructor as it's always the same information and didn't see why the defaults object has to be created every time a new DelimiterStream instance is made.